draft-ietf-tls-tls13.md

```diff
@@ -1991,6 +1993,37 @@ willing to negotiate TLS 1.2 MUST behave in accordance with the requirements of
   TLS 1.2 is negotiated. In TLS 1.2, RSASSA-PSS is used with RSA cipher suites.
 
 
+#### Certificate Authorities
+
+The "certificate_authorities" extension is used to indicate the
+certificate authorities which an endpoint supports and which SHOULD
+be used by the receiving endpoint to guide certificate selection.
+
+The body of the "certificate_authorities" extension consists of a
+CertificateAuthorities structure.
+
+%%% Server Parameters Messages
+
+       opaque DistinguishedName<1..2^16-1>;
+
+       struct {
+           DistinguishedName authorities<3..2^16-1>;
+       } CertificateAuthoritiesExtension;
+
+authorities
+: A list of the distinguished names {{X501}} of acceptable
+  certificate authorities, represented in DER-encoded {{X690}} format.  These
+  distinguished names may specify a desired distinguished name for a
+  root CA or for a subordinate CA; thus, this message can be used to
+  describe known roots as well as a desired authorization space.
+{:br}
+
+The client MAY send the "certificate_authorities" extension in the ClientHello
+message. The server MAY send it in the CertificateRequest message.
+
+The "trusted_ca_keys" extension, which serves a somewhat similar
+purpose {{RFC6066}}, but is more complicated, is not used in TLS 1.3.
```

Comment:
The server can send it in CertificateRequest so the client can pick a cert to send back?
(But I'm not sure I'm answering the right question.)